db/schema.rb

```diff
@@ -445,6 +448,12 @@
     t.integer "created_by_id"
   end
 
+  create_table "task_notifications", force: :cascade do |t|
+    t.bigint "measure_id", null: false
+    t.datetime "created_at", null: false
+    t.index ["measure_id"], name: "index_task_notifications_on_measure_id"
+  end
```

Comment:
do we really need another table here? I assumed the send queue and handling would happen elsewhere

Reply:
haha I guess none of these =)